src/app/app-services/AuthInterceptor.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable, Inject } from '@angular/core';
+import { HttpInterceptor, HttpHandler, HttpRequest, HttpEvent } from "@angular/common/http";
+import { Observable } from 'rxjs';
+import { AuthorizationService } from './authorization.service';
+
+@Injectable()
+export class AuthInterceptor implements HttpInterceptor {
+    constructor(
+        private readonly authService: AuthorizationService,
+        @Inject('Storage') private readonly localStorage: Storage) { }
+    intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
```

Comment:
Just as a tip, here's how I usually format the code when a method has many arguments:

```typescript
constructor(
    private readonly authService: AuthorizationService,
    @Inject('Storage') private readonly localStorage: Storage
)  { }

intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
    // code
}
```

Reply:
Thanks for the tip, I was not sure about how to format all this params.